src/tests/api/users/update/validate.spec.ts

```
@@ -0,0 +1,201 @@
+import { validate } from "api/users/update/validate";
+import { StatusCodeEnum } from "enums/status-code";
+import { CustomError } from "utils/error";
+
+describe("create validate", () => {
+	const validId = "af93d117-4dfe-40c9-9456-f0db998b4309";
+	const validDisplayName = "DisplayName Teste";
+	const validBio = "Bio Teste";
+
+	describe("Successfully", () => {
+		it("should return validated when suplied params", async () => {
+			let result: any;
+
+			try {
+				result = await validate({
+					id: validId,
+					displayName: validDisplayName,
+					bio: validBio,
+				});
+			} catch (err: any) {
+				result = err;
+			}
+
+			expect(result).toStrictEqual({
+				id: validId,
+				displayName: validDisplayName,
+				bio: validBio,
+			});
+		});
+	});
+
+	describe("Invalid params", () => {
+		it("should throw an error if the id its a string, but not a uuid", async () => {
+			let result: any;
+
+			try {
+				result = await validate({
+					id: "5" as any,
+				});
+			} catch (err) {
+				result = err;
+			}
+
+			expect(result instanceof CustomError).toBeTruthy();
+			expect(result.statusCode).toBe(StatusCodeEnum.BAD_REQUEST);
+			expect(result.message).toBe("id must be a valid UUID");
+		});
+	});
+
+	describe("Invalid type params", () => {
+		it("should throw an error if the id has the wrong type", async () => {
```

`"should throw an error with invalid id type"`